ffv1.md

```diff
@@ -1009,6 +1014,10 @@ Architecture overview of slices in a `Frame`:
 
 ## Slice
 
+A `Slice` is an independent part of a frame. `Slice` are useful for multithreaded decoding.
```

Comment:
Grammar here is a bit off. Also I think the definition of Slice within the Definition section at the beginning is better. Rather than define it twice perhaps the one in the Definitions section could be moved here. For the second sentence, maybe better to say like "The use of multiple `Slices` per frame can be useful for taking advantage of the opportunities of multithreaded encoding and decoding."